.goreleaser.yml

```diff
@@ -102,6 +102,11 @@ dockers:
   image_templates:
   - 'ghcr.io/kanisterio/cassandra:{{ .Tag }}'
   dockerfile: 'docker/cassandra/Dockerfile'
+- ids:
+  - kando
+  image_templates:
+  - 'ghcr.io/kanisterio/mysql:{{ .Tag }}'
```

Comment:
I think we should name this image, `mysql-kopia` or `mysql-tools` so that when we use this image while installing mysql, it would be clearer that we are using this image because it has kopia.

Reply:
I think we're doing similar thing with some other images like cassandra where we're incorporating restic with original base image and it's named cassandra only, so I think we could go with mysql only.

Reply:
mmm, but do you agree that having the name just `mysql` would confuse others as to why are we building the mysql image and not using the one that is provided with chart.
If you agree, then we should change this, even though we have already done similar things with Cassandra.

Reply:
Yeah I Agree ! changed the name